portal/server/routes/contract/loan/index.js

```diff
@@ -271,8 +305,34 @@ router.post('/contract/:_id/loan/:loanId/save-go-back', provide([LOAN]), async (
   const { _id: dealId, loanId, userToken } = requestParams(req);
   const { loan } = req.apiData.loan;
 
-  let modifiedBody = handleNameField(req.body);
-  modifiedBody = premiumFrequencyField(req.body, loan);
+  const allowedProperties = [
```

Comment:
I could build this from the other 3 endpoints as it's the 3 other arrays concatenated and use some constants as the allowed properties outside the route methods to make this a bit more maintainable